src/Hazelcast.Net/Clustering/ClusterConnections.cs

```diff
@@ -120,10 +127,22 @@ await foreach(var (member, token) in _clusterMembers.MembersToConnect.WithCancel
                 else
                 {
                     var details = wasCanceled ? "canceled" : "failed";
```

Comment:
Is it still needed to set `details` as canceled or failed here?

Reply:
well, yes, I think so? `exception` could be `null`?

Reply:
I mean in every block `details` var is set to something else, are we using the values `canceled` or `failed` after setting it?

Reply:
Yes but if `exception` is null? None of the blocks run, or am I missing something?

Reply:
Ah sorry my bad, i missed the `details` at the end of the line